src/sentry/models/organization.py

```diff
@@ -81,6 +81,7 @@ class Organization(Model):
 
     flags = BitField(flags=(
         ('allow_joinleave', 'Allow members to join and leave teams without requiring approval.'),
+        ('enhanced_privacy', 'Enable enhanced privacy controls to limit personally identifiable information (PII) as well as source code in things like emails.'),
```

Comment:
"in things like emails" might be better worded as "in notifications" if we plan to ensure this is respected by plugins.